src/modules/ecdsa_adaptor/dleq_impl.h

```diff
@@ -0,0 +1,158 @@
+#ifndef _SECP256K1_DLEQ_IMPL_H_
```

Comment:
```suggestion
#ifndef SECP256K1_DLEQ_IMPL_H
```
(identifiers starting with `_` and an uppercase letter are reserved in C)